Core/WebViewController.swift

```diff
@@ -462,7 +462,7 @@ extension WebViewController: WKNavigationDelegate {
     private func reportHttpsUpgradeSiteError(url: URL, error: String) {
         let params = [
             Pixel.EhdParameters.errorCode: error,
-            Pixel.EhdParameters.url: url.simpleUrl
+            Pixel.EhdParameters.url: "https://\(url.host ?? "")"
```

Comment:
How I'm reading this, the default case is `https://` if `url.host` isn't defined?

Reply:
In reality it will be defined (we have it as blank in the current logic too) but the host is optional so I need to provide a fallback. What would you prefer the default to be?

Reply:
I'd just prefer the whole operation not happen without the required info.

Reply:
@nilnilnil I added a guard, let me know if you think it needs to be tweaked more